Conrollers/postConroller.js

```diff
@@ -10,5 +10,14 @@ const postArticle = (req, res, next) => {
       next(err);
     });
 };
-
-module.exports = { postArticle };
+function patcharticle_id(req, res, next) {
```

Comment:
i would give your functions more descriptive names, like what is this doing? It's updating the votes in the article so maybe something like `updateVotes` would be nice and descriptive here